app/overlays/mtl/dts_overlay.conf

```diff
@@ -1,3 +1,8 @@
+# Use DTS pre-built library for production
+CONFIG_COMP_STUBS=n
+# Google RTC library is WIP
+CONFIG_GOOGLE_RTC_AUDIO_PROCESSING_MOCK=y
```

Comment:
Is this needed for a topology?

Reply:
Ah yes since GOOGLE_RTC_AUDIO_SUPPORT is set enabled on all chrome topology builds:
https://github.com/thesofproject/sof/blob/main/tools/topology/topology2/production/tplg-targets-ace1.cmake#L72

I wonder that we should make the production build by removing GOOGLE_RTC_AUDIO_SUPPORT from topology. We don't expect a mock module running on production.

Reply:
it is required, however, for check compilation and for CI.
Google AEC shimm won't compile (to be precise: linker will fail) unless either a mock or an AEC library itself is linked.
So - we can eihter 
 - turn off compilation of google_rtc completely (not reccomended!) 
 - use MOCK
 - upstream an AEC binary